src/coreclr/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.CoreCLR.cs

```diff
@@ -615,6 +630,28 @@ public TypeHandle GetArrayElementTypeHandle()
         public extern uint GetNumInstanceFieldBytes();
     }
 
+    // Subset of src\vm\methodtable.h
+    [StructLayout(LayoutKind.Explicit)]
+    internal unsafe struct MethodTableAuxiliaryData
```

Comment:
I'm not sure if this is the desired direction to expose this, but it's relatively simple.